lntest/itest/lnd_channel_force_close.go

```diff
@@ -187,6 +186,13 @@ func testCommitmentTransactionDeadline(net *lntest.NetworkHarness,
 		"expected fee rate:%d, got fee rate:%d", maxPerKw, feeRate,
 	)
 
+	// Since the sweep transaction needs to spend from an unconfirmed
```

Comment:
Further checked and I think I got it wrong. Yeah there's no guarantee in either we'd sweep the local or the remote first. Then this is not a test flake as we cannot reliably use the anchor tx to CPFP our closing tx with neutrino backend. I'll dig into it a bit more later.
Meanwhile, do you think [this](https://github.com/lightningnetwork/lnd/pull/5549/commits/c34d0745b012d09ce1ec9e932e5f619aa8d35b12) could be a fix in terms of the conflicting inputs (haven't followed that PR...)?

Reply:
I think we can reliably use the anchor output to CPFP our closing TX. Even without the fix in https://github.com/lightningnetwork/lnd/pull/5549/commits/c34d0745b012d09ce1ec9e932e5f619aa8d35b12. We just need to be aware that there will always be an orphan TX for the remote anchor in Neutrino, since we can't invalidate that TX (I think). So I circumvented that problem in #5509 by making sure we had enough UTXOs for sweeping so that the one orphaned UTXO isn't the only one we have.
But maybe I don't fully understand the fix in https://github.com/lightningnetwork/lnd/pull/5549/commits/c34d0745b012d09ce1ec9e932e5f619aa8d35b12 either...

Reply:
> I think we can reliably use the anchor output to CPFP our closing TX.
Yeah I think we are fine for a full node, not the case in Neutrino tho. As we cannot guarantee there's always an extra utxo that can be used for CPFP the remote tx, thus it might fail in Neutrino (I think)? 
Think I need to take some time to read that PR to understand what's going on with our sweeper😂